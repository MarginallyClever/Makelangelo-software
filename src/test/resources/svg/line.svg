<svg
  xmlns="http://www.w3.org/2000/svg"
  viewBox="0 0 163.3 47.2"
  xmlns:v="https://vecta.io/nano"
>
  <path
    d="M163.1.5L.1 46.7"
    fill="none"
    stroke="#e62227"
    stroke-miterlimit="10"
  />
</svg>