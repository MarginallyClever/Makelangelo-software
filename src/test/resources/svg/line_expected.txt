[TOOL R230 G34 B39 A255 D1.000, TRAVEL X0.999 Y-0.359, DRAW_LINE X0.001 Y-0.641]